tests/AbstractTest.php

```diff
@@ -22,12 +23,18 @@ abstract class AbstractTest extends TestCase
      */
     public $httpClient;
 
+    /**
+     * @var ExpressionLanguage
+     */
+    protected $expressionLanguage;
+
     /**
      * {@inheritdoc}
      */
     protected function setUp()
     {
         $this->httpClient = new Client();
+        $this->expressionLanguage = new ExpressionLanguage();
```

Comment:
We could [register here functions](https://symfony.com/doc/current/components/expression_language/extending.html#registering-functions) that we might use in the assertions (see comment below for more info).

We can easily register PHP function in the following way:

```php
        $this->expressionLanguage = new ExpressionLanguage();
        $this->expressionLanguage->addFunction(ExpressionFunction::fromPhp('count'));
```

Reply:
Fixed.